### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

### DISCUSSION THREAD

**Discussion by altacountbabi** [community/community#13812] [Copilot Conversations]
says on the website im good but vscode doesnt?

**Comment by davecheney** [comment]
Are you using VSCodium or OSS Code?

**Reply by dev-eth** [reply]
@altacountbabi 
After installing the Github copilot extension, did you authorize it with Github? Make sure you already signed to your GitHub.
Also, you need to wait for the invitation of Github Copilot Technical Preview.
For me, it took 1 day.

**Reply by altacountbabi** [reply]
it doesnt ask me about authorization it just stays on for like 1 second and it turns off im logged in with this account

![image](https://user-images.githubusercontent.com/82091823/160822266-22c38d2b-271b-4554-85b8-93db5cfc4a66.png)

**Reply by davecheney** [reply]
Please double check that you have VSCode installed, not VSCodium or OSS Code.

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by dev-eth** [reply]
Go to the VSCode extension list & check if the Copilot extension has the "Reload required" button.
If yes, click it.

**Reply by altacountbabi** [reply]
> Go to the VSCode extension list & check if the Copilot extension has the "Reload required" button. If yes, click it.

I already did that and there isnt one

**Reply by altacountbabi** [reply]
> Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

![image](https://user-images.githubusercontent.com/82091823/160831008-e84b7a59-077e-4bf4-a91e-b8ad8f9c6708.png)

**Reply by altacountbabi** [reply]
> Please double check that you have VSCode installed, not VSCodium or OSS Code.

its vscode

**Reply by dev-eth** [reply]
It looks like you are still on the waitlist, can you double-check if you have the invitation?

**Reply by altacountbabi** [reply]
> It looks like you are still on the waitlist, can you double-check if you have the invitation?

![image](https://user-images.githubusercontent.com/82091823/160834623-91aca130-35b9-4755-ab85-afcd01b924cc.png)
I already did and it says its enabled

**Reply by dev-eth** [reply]
![Screen Shot 2022-03-30 at 8 31 34 AM](https://user-images.githubusercontent.com/66653006/160835139-621decc8-fea1-4442-b0c9-43ff7bd4d9b8.png)

Did you receive this email from the github?

**Reply by altacountbabi** [reply]
> ![Screen Shot 2022-03-30 at 8 31 34 AM](https://user-images.githubusercontent.com/66653006/160835139-621decc8-fea1-4442-b0c9-43ff7bd4d9b8.png)
> 
> Did you receive this email from the github?

yes
![image](https://user-images.githubusercontent.com/82091823/160835338-1cafc0ad-9c87-40ac-91ef-ac6332daa036.png)

**Comment by dev-eth** [comment]
I signed up for the Github copilot program yesterday, & received the invitation email just before.
Copilot is a very amazing feature, can't wait to see more updates from it.
Thanks, @github team.
